projects/wpdx/src/app/common-components/photo-gallery/photo-gallery.component.less

```diff
@@ -0,0 +1,56 @@
+@import '../common.less';
+@import '../popup-contents.less';
+
+mat-icon {
+    cursor: pointer;
+    position: relative;
+    top: -200px;
+    top: -250px;
+    background-color: whitesmoke;
+    border-radius: 30%;
+}
+.forward {
+    left: 300px;
+}
+
+img {
+    // margin-left: 16px;
+    // margin-right: 8px;
+    margin-top: 32px;
+    width: 100%;
+    height: 500px;
+    flex: 0 0 auto;
+    align-self: center;
+    .nice-shadow;
+}   
+
+.photo-gallery{
+    position: relative;
+    height: 500px;
+    display: block;
+    overflow: hidden;
+}
+
+.current-photo{
+    position: absolute;
```

Comment:
don't repeat css properties (position, top, transition) but move them to a shared section.